backend/src/test/java/de/neuefische/backend/service/RestCountriesServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package de.neuefische.backend.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.neuefische.backend.exception.RestCountryException;
+import de.neuefische.backend.model.Country;
+import de.neuefische.backend.model.restcountries.RestCountry;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RestCountriesServiceTest {
+
+    RestCountriesService restCountriesService = new RestCountriesService();
+
+    @Test
+    void refactorApiResponse_deseralize_properly() throws JsonProcessingException, RestCountryException {
```

Comment:
You are only testing the mapping from an array to a list and not the deserialization this happens in your other class within the webclient